app/views/restaurant/show.html.erb

```diff
@@ -8,11 +9,19 @@
       </ul>
     </section>
 
-<iframe
-  width="600"
-  height="450"
-  frameborder="0" style="border:0"
-  src="https://www.google.com/maps/embed/v1/directions?key=<%=ENV['GOOGLE_API_KEY']%>
-    &origin=<%=@current_user.lat%>,<%=current_user.lng%>
-    &destination=<%=@restaurant.lat%>,<%=@restaurant.lng%>" allowfullscreen>
-</iframe>
+  <iframe
+    width="600"
+    height="450"
+    frameborder="0" style="border:0"
+    src="https://www.google.com/maps/embed/v1/directions?key=<%=ENV['GOOGLE_API_KEY']%>
+      &origin=<%=@current_user.lat%>,<%=current_user.lng%>
+      &destination=<%=@restaurant.lat%>,<%=@restaurant.lng%>" allowfullscreen>
+  </iframe><br><br>
+Not Satisfied? <br>
+  <a class="waves-effect waves-light btn-small">
```

Comment:
Added buttons to refresh the page (which is our re-shake button) as well as a button to take you back to adjust your parameters.